setup.py

```diff
@@ -100,7 +100,7 @@ def build_extension(self, ext):
             "mypy==0.942",
             "pre-commit==2.17.0",
         ],
-        "test": ["pytest", "pytest-cov", "hilbertcurve==1.0.5"],
+        "test": ["pytest", "pytest-cov", "hilbertcurve==1.0.5", "hypothesis==6.54.6"],
```

Comment:
Note: not essential, but I found property testing useful for getting the `cyclic_difference` function correct. Open to removing